common/src/WooCommerce.php

```diff
@@ -123,23 +132,52 @@ public function order_completed($order_id) {
 
         // send invite
         $api = new API($api_domain, $shop_id, $api_key);
+
         try {
             $api->invite($data);
-        } catch (WebwinkelKeurAPIAlreadySentError $e) {
-            // that's okay
         } catch (WebwinkelKeurAPIError $e) {
-            $wpdb->insert($this->plugin->getInviteErrorsTable(), [
-                'url'       => $e->getURL(),
-                'response'  => $e->getMessage(),
-                'time'      => time(),
-            ]);
+            $this->logApiError($e);
             $this->insert_comment(
                 $order_id,
                 sprintf(
                     __('The %s invitation could not be sent.', 'webwinkelkeur'),
                     $this->plugin->getName()
                 ) . ' ' . $e->getMessage()
             );
+            return;
+        }
+
+        $this->insert_comment(
+            $order_id,
+            sprintf(
+                __('An invitation was sent to %s dashboard.', 'webwinkelkeur'),
+                $this->plugin->getName()
+            )
+        );
+    }
+
+    public function addGtinOption() {
+        if (GtinHandler::getActivePlugin() || !get_option($this->plugin->getOptionName('product_reviews'))) {
+            return;
+        }
```

Comment:
Hmm...  So we're still adding our own GTIN field if the user has selected an attribute/meta key to use?